packages/angular_devkit/build_angular/src/webpack/configs/typescript.ts

```diff
@@ -51,20 +51,20 @@ function createIvyPlugin(
     }
   }
 
-  let inlineStyleMimeType;
+  let inlineStyleFileExtension;
```

Comment:
```suggestion
  const inlineStyleFileExtension = buildOptions.inlineStyleLanguage ?? 'css';
```

Reply:
The current code and the various suggestions all suffer from a similar problem in that they conflate the semantic meaning of the language with the language's file extension (as used within the Webpack configuration).

I have an additional refactor on top of this one that retrieves style language information from an adjusted form of the language info structure in the style configuration file (https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/webpack/configs/styles.ts#L268).  That would in turn change this code into a lookup of language info and ensure that the file extension used matches the Webpack configuration's expected value.
If preferred, I can include that commit in this PR as well.

Reply:
Lets do that.